connector_salesforce/README.rst

```diff
@@ -0,0 +1,247 @@
+  Callback URL --> public_odoo_url/salesforce/oauth
+
+
+Once done, you have to manage your app and ensure the
+`Refresh token is valid until revoked` parameter is set
+and configure the desired authorization of the app.
+
+In Odoo
+#######
+
+Go to the menu `Connector -> Salesforce -> Salesforce backend` and create a new backend.
+Go to the Authentication Pane of your backend and enter the following parameters:
+
+ - `Authentication Method`: Choose oauth2
+ - `URL`: Enter the URL of your Salesforce instance
+ - `Connect on sandbox instance`: Check it if you are working on a sandox Salesforce instance
+ - `OAuth2 Consumer Key`: Enter the consumer key provided by Salesforce app
+ - `OAuth2 secret`: Enter the consumer secret provided by Salesforce app
+ - `Public secure URL of Odoo (HTTPS)`: Enter the callback URL you have enterd by Salesforce
+ - `OAuth2 client authorization code`: Enter the authorization code provided by Salesforce app
+
+Choose a default shop in `Import/Export Configuration` pane and save your backend.
+
+Then simply press `the Authorize Odoo in Salesforce` and follow the instruction provided
+in Salesforce. Once done, your application must be authorized in Salesforce and tokens
+must be present in backend.
+
+!!!Only the OAuth flow is production proof. Other Authentication methods are still exprimental!!!
+
+
+User Password flow
+******************
+
+This flow allows a user to connect to api using SOAP access
+in order to get a token. This approach is simpler but less secure.
+The first step is to pass the domain of your Salesforce instance
+and an access token straight in Odoo backend.
+
+You must have the full URL e.g (https://na1.salesforce.com) of your instance.
+
+There are also two means of authentication:
+ - Using username, password and security token
+ - Using IP filtering, username, password and organizationId
+
+Simply choose corresponding authentication method and
+set required fields.
+
+
+Configure Import Export with Salesforce
+---------------------------------------
+
+Now your backend is set and open in the `Import/Export Configuration` pane
+
+Products Synchronisation
+************************
+
+  The first step is to choose the Direction of the product synchronisation. Once set you should not change it.
+
+**  There is the possibility to do an initial import of the product and then change the direction **
+
+Product Type Mapping
+********************
+
+The product type of Salesforce can not be imported and mapped as is.
+You have to manually set the mapping under the `Product Configuration` section.
+
+Simply choose the Odoo type and enter the corresponding name of the Salesforce type.
+
+
+Price Book mapping
+##################
+
+The currencies of Salesforce can not be imported and mapped as is.
+You have to manually set the mapping under the `Product Book Entry Configuration` section.
+
+Simply choose an Odoo pricelist version and map it with a Currency ISO code available in
```

Comment:
Simply select an Odoo pricelist version with desired currency and link it with a Currency ISO code available in
Salesforce.